mud-contracts/world/src/codegen/world/ISmartGate.sol

```diff
@@ -42,5 +43,5 @@ interface ISmartGate {
 
   function eveworld__isGateLinked(uint256 sourceGateId, uint256 destinationGateId) external view returns (bool);
 
-  function eveworld__calculatDistance(uint256 sourceGateId, uint256 destinationGateId) external view returns (uint256);
+  function eveworld__isWithinRange(uint256 sourceGateId, uint256 destinationGateId) external view returns (bool);
```

Comment:
```suggestion
  function eveworld__isGateWithinRange(uint256 sourceGateId, uint256 destinationGateId) external view returns (bool);
```

after using `eveworld__inProximity` and `eveworld__agression` I think we should be a bit more verbose

Reply:
Yes good catch, Will recompile it and push. Thanks